File: dintinct-lenguage/AreaTriangulo.cpp
#include <iostream>
int main () {
  //definir valotres
  printf("Ejercicio 01: Area de un Triangulo\n");
  int b=0, h=0, area=0;
  //datos de entarda
  printf("Ingrese base:");
  scanf("%i", &b);
  printf("Ingrese altura");
  scanf("%i", &h);
  //proceso
  area=(b*h)/2;
  //dartos de ssalida
  printf("Area de triangulo es:%i %s", area, "\n");
  return 0;
}

File: dintinct-lenguage/AreaTriangulo.dart
import "dart:io";
void main() {
//definir varibles u otros
print("Ejercio 01: Area Triangulo");
//datos de entrada
print("Ingrese base:");
var b=int.parse(stdin.readLineSync());
print("Ingrese altura:");
var h=int.parse(stdin.readLineSync());
//proceso
var area=(b*h)/2;
//salida
print("El area del triangulo es: $area");
}